internal/uvm/create.go

```diff
@@ -286,23 +315,46 @@ func Create(opts *UVMOptions) (*UtilityVM, error) {
 		}
 
 		if opts.ConsolePipe != "" {
-			hcsDocument.VirtualMachine.Chipset.UEFI.BootThis.OptionalData += " console=ttyS0,115200"
-			hcsDocument.VirtualMachine.Devices.COMPorts = &schema2.VirtualMachinesResourcesComPortsV2{Port1: opts.ConsolePipe}
+			vmDebugging = true
+			kernelArgs += " console=ttyS0,115200"
+			vm.Devices.COMPorts = &schema2.VirtualMachinesResourcesComPortsV2{Port1: opts.ConsolePipe}
 		}
 
 		if opts.EnableGraphicsConsole {
-			hcsDocument.VirtualMachine.Chipset.UEFI.BootThis.OptionalData += " console=tty"
-			hcsDocument.VirtualMachine.Devices.Keyboard = &schema2.VirtualMachinesResourcesKeyboardV2{}
-			hcsDocument.VirtualMachine.Devices.Rdp = &schema2.VirtualMachinesResourcesRdpV2{}
-			hcsDocument.VirtualMachine.Devices.VideoMonitor = &schema2.VirtualMachinesResourcesVideoMonitorV2{}
+			vmDebugging = true
+			kernelArgs += " console=tty"
```

Comment:
I cant quite tell from the documentation. Is `console=` multi argument? Or would `opts.ConsolePipe == "anything" && opts.EnableGraphicsConsole`  result in something that doesn't output for either correctly?

Reply:
Yeah, it's multi argument.

Reply:
Ok deal.